### PR TITLE
feat: include breaking change message if provided

### PR DESCRIPTION
**Pull request by MarkMcCulloh**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Adding a `BREAKING CHANGES: ` trailer, per conventional commits, will now use that message in the "Breaking Changes" section of the changelog.

### 📝 Checklist

- [ ] I have linked an issue or discussion.
  - Related to #112 but doesn't resolve it because there should be some documentation updates (see below)
- [ ] I have updated the documentation accordingly.
  - Not sure where the documentation is supposed to go?

### CODE REVIEW

File: src/markdown.ts [hunk 1]
```diff
@@ -176,8 +176,8 @@ function formatName(name = "") {
     .join(" ");
 }
 
-function groupBy(items: any[], key: string) {
-  const groups = {};
+function groupBy<TItem>(items: TItem[], key: string) {
+  const groups: Record<string, TItem[]> = {};
```
Comment (MarkMcCulloh) [inline src/markdown.ts:180]: This function was returning `any` for me locally, seemed useful to provide types

**Comment by reslear** [comment]
any changes?

**Comment by pi0** [comment]
Thank you for this PR and sorry took long to review. Support is added via more recent https://github.com/unjs/changelogen/pull/228

**Comment by Maxel01** [comment]
@pi0 the support for the indicator has been added, but I think it is still not possible to add the message behind BREAKING CHANGE: to the changelog. See also #112

**Pull request closed without merging**